.gitlab/pipeline/zombienet.yml

```diff
@@ -9,6 +9,7 @@
     RUN_IN_CI: "1"
     KUBERNETES_CPU_REQUEST: "512m"
     KUBERNETES_MEMORY_REQUEST: "1Gi"
+    NODE_OPTIONS="--max_old_space_size=4096"
```

Comment:
The docs [here](https://nodejs.org/api/cli.html#node_optionsoptions) mention it with a dash. Does it make a difference?

Reply:
> The docs [here](https://nodejs.org/api/cli.html#node_optionsoptions) mention it with a dash. Does it make a difference?

Yes, that correct. Thanks for the catch 🙌🙌. I will change it asap.

Reply:
Just as reference, https://github.com/paritytech/polkadot-sdk/pull/6139#discussion_r1808449528. Thanks!